lib/modules/manager/mix/utils.ts

```diff
@@ -0,0 +1,42 @@
+import type { Registry } from './types';
+
+/**
+ * Resolves alias in repository string.
+ *
+ * @param repository to be resolved string
+ * @param registryAliases Records containing registryAliases as key and to be resolved URLs as values
+ *
+ * @returns  resolved alias. If repository does not contain an alias the repository string will be returned. Should it contain an alias which can not be resolved using `registryAliases`, null will be returned
+ */
+export function resolveAlias(
+  packageName: string,
+  registryAliases: Record<string, string>,
+): string | null {
+  if (!isAlias(packageName)) {
+    return packageName;
+  }
+
+  const [, repoName, ,] = packageName.split(':');
```

Comment:
```suggestion
  const [, repoName] = packageName.split(':');
```

Reply:
I don't understand the suggestion. This wants to extract the second value in whatever is returned.